### DISCUSSION TITLE
Nach Update WebUi nicht erreichbar

### DISCUSSION THREAD

**Discussion by BZBerlin** [OpenCCU/OpenCCU#2290] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Nach Automatischem Update ist die WebUi nicht mehr erreichbar.
Rasberry ist anpingbar.
Letzte Meldung bei der Installation war -- Verbindung unterbrochen.

Kann mir jemand bei diesem Problem helfen bin selbst nicht Linux-fest und brüchte ggf. "Kochrezept" .
Habe Sicherungscopy vom System.

### Describe the behavior you expected

Habe nach ca. 1/2 Stunde den Raspi neu gestartet.
Keine Besserung

### Steps to reproduce the issue

1Bin wie gewohnt auf automatisches update und bin den Anweisungen gefolgt.
2.
3.
...

### What is the version this bug report is based on?

3.67.10.202330225

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
Leider nicht
```

**Comment by jens-maus** [comment, marked as answer]
Da dies kein allgemeines Problem/Bug/Issue sein **kann**, habe ich dies mal in den Diskussionsbereich verschoben.

Wie immer bei diesen "Update-Probleme" ist auch hier die erste Vermutung es wird an einer sterbenden SD-Karte liegen. Ergo: In den Müll damit, frische aus der Packung nehmen und frisch mit RaspberryMatic bespielen und dann das letzte lauffähige Backup einspielen.

**Reply by BZBerlin** [reply]
Bin genauso vorgegangen wie empfohlen. Habe alle dazu nötigen Hilfsmittel im Forum gefunden.
Habe aber RedMatic nachinstallieren müssen, war nicht im Backup. Allerdings die Daten waren nach der Neuinstallation noch da.
Trotzdem vielen Dank für die Mühe, aber bei so einer Situation steht man erstmal mit gesenktem Kopf da. Habe fast so ziemlich alles in meinem Hause über den Raspi gesteuert.

**Reply by jens-maus** [reply]
> Habe fast so ziemlich alles in meinem Hause über den Raspi gesteuert.

Dann würde ich empfehlen über einen Weggang von SD Karten hin zu eMMC basiertem Medium nachzudenken. Damit gibt es recht gute Erfahrungen und sollte die Lebensdauer ziemlich erhöhen helfen:

https://www.berrybase.de/raspikey-plug-play-emmc-modul-fuer-raspberry-pi-32gb
